tests/marine/test_marine.py

```diff
@@ -388,3 +388,13 @@ def test_filter_and_parse_with_no_parameters(
 ):
     with pytest.raises(ValueError, match="must be passed"):
         marine_instance.filter_and_parse(tcp_packet)
+
+
+def test_get_epan_auto_reset_count_return_default_value(marine_instance: Marine):
+    assert marine_instance.epan_auto_reset_count == 20000
+
+
+def test_set_epan_auto_reset_count_return_default_value(marine_instance: Marine):
+    SOME_VALUE = 1
+    marine_instance.epan_auto_reset_count = SOME_VALUE
+    assert marine_instance.epan_auto_reset_count == SOME_VALUE
```

Comment:
I think the convention is `expected == actual`, but I'll mark it as approved and leave it up to @yehonatanz to decide